Formula/h/heppdt2.rb

```diff
@@ -1,8 +1,8 @@
 class Heppdt2 < Formula
   desc "Translation for particle ID's to and from various MC generators and PDG standard"
   homepage "https://cdcvs.fnal.gov/redmine/projects/heppdt/wiki"
-  url "https://cern.ch/lcgpackages/tarFiles/sources/HepPDT-2.06.01.tar.gz"
-  sha256 "12a1b6ffdd626603fa3b4d70f44f6e95a36f8f3b6d4fd614bac14880467a2c2e"
+  url "https://cern.ch/lcgpackages/tarFiles/sources/HepPDT-3.04.01.tar.gz"
```

Comment:
should we create `heppdt3` instead?

Reply:
Hmm, good point. Would it be recommended in this case to just duplicate this all to a new `Heppdt3` class formula?

Side note is that the maintenance status of this software is also a bit dubious – the homepage is not public anymore and I don't see a source repository that's public either (for example Cern's [GitLab](https://gitlab.cern.ch/) requires a login). Just this one tarball that's visible via a directory listing. But, if it still works 🤷‍♂️ I see there's a case where it might be decided to deprecate due to unmaintained status, but I can't really confirm if it's truly unmaintained either. Any advice here welcome too.

Reply:
Yeah, I would say so

Or maybe better go with `heppdt@2` and `heppdt`?

cc @Homebrew/core for thoughts.

Reply:
If v2.x is supported upstream, and we have an actual need for it we can create a versioned formula. Otherwise we should just rename this to `heppdt`. It looks like [that's what other packagers](https://repology.org/projects/?search=heppdt) have it as.

Reply:
Maintenance status is definitely questionable. v3.04.01 is from 2009-11-24. Other online references are steadily being removed after homepage went down.

There was some minor revisions in 2020 before site became inaccessible - https://web.archive.org/web/20210318015301/https://cdcvs.fnal.gov/redmine/projects/heppdt/repository

Only CERN reference I can find is a download link on HepMC page https://hepmc.web.cern.ch/hepmc/#heppdt_releases but no useful info about the project.

Reply:
> the homepage is not public anymore and I don't see a source repository that's public either

This sounds worth deprecating.



> If v2.x is supported upstream, and we have an actual need for it we can create a versioned formula. Otherwise we should just rename this to `heppdt`. It looks like [that's what other packagers](https://repology.org/projects/?search=heppdt) have it as.

Agreed.

Reply:
Thanks for the feedback! Let me know if this sequence of actions seems reasonable?

1. PR to [rename it](https://docs.brew.sh/Rename-A-Formula) to `heppdt`
2. PR to deprecate the formula (not sure the recommended timeline for that, but there are very few installs, so.. 6 months? 0 months?)
3. Optionally update this 158982 PR to bump the version up to this 3.x version on the `heppdt` formula

Reply:
@timsutton You're very welcome, thanks for being to receptive to it ❤️!

I would suggest that if we decide to deprecate it it happens immediately and we don't bother with renaming/updating.

Reply:
@MikeMcQuaid Makes total sense. I'll set that up in https://github.com/Homebrew/homebrew-core/pull/159069 and close this PR.